Mathlib/LinearAlgebra/Matrix/SpecialLinearGroup.lean

```diff
@@ -490,4 +490,10 @@ theorem T_mul_apply_one (g : SL(2, ℤ)) : ↑ₘ(T * g) 1 = ↑ₘg 1 := by
 theorem T_inv_mul_apply_one (g : SL(2, ℤ)) : ↑ₘ(T⁻¹ * g) 1 = ↑ₘg 1 := by
   simpa using T_pow_mul_apply_one (-1) g
 
+lemma S_mul_S_eq : (↑ₘS * ↑ₘS) = -1 := by
```

Comment:
Does this belong here?

Reply:
I thought this was the natural place to put it since it where there are other results about S and T, but I'm happy to move it if you think there's a better file to put it in.

Reply:
Hm, not sure why I thought it could go elsewhere. Never mind.